meinberlin/assets/scss/components/_select_dropdown.scss

```diff
@@ -18,9 +18,11 @@
     margin-right: 0.6em;
 }
 
-.select-dropdown__item:first-child {
-    padding-top: 0.8em;
-    padding-bottom: 0.8em;
-    color: $text-color-gray;
-    background-color: $bg-secondary;
+.select-icon {
```

Comment:
not sure why this is necessary. but i probably broke it while renaming the css classes in core